src/ResourceManager/KeyVault/Commands.KeyVault/Commands/ManagedStorageAccounts/UpdateAzureKeyVaultManagedStorageAccountKey.cs

```diff
@@ -0,0 +1,87 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.KeyVault.Models;
+using System.Globalization;
+using System.Management.Automation;
+using KeyVaultProperties = Microsoft.Azure.Commands.KeyVault.Properties;
+
+namespace Microsoft.Azure.Commands.KeyVault
+{
+    [Cmdlet( VerbsData.Update, CmdletNoun.AzureKeyVaultManagedStorageAccountKey,
+        SupportsShouldProcess = true,
+        ConfirmImpact = ConfirmImpact.High,
```

Comment:
@markcowl: this cmdlet in effect changes the account key on the storage account, and so it can cause immediate disruption of service. As I understand, there were background discussions with you and Kunal re: finding a more appropriate verb. We must retain the 'high impact' semantics of this cmdlet, and the ability to force a change.